### PR TITLE
complete project 0.1

### PR DESCRIPTION
**Pull request by ndikintum**

### CODE REVIEW

File: assets/main.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  const chessboard = document.getElementById('chessboard')
+  const checkButton = document.getElementById('checkbutton')
+  const resultElement = document.getElementById('result')
+  let selectedQueens = []
+
+  function canQueensAttack (queen1, queen2) {
+    const [x1, y1] = queen1
+    const [x2, y2] = queen2
+
+    return (
```
Comment (DanCarl857) [inline assets/main.js:11]: This is very naz ;-)

File: assets/main.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        chessboard.appendChild(button)
+      }
+    }
+  }
+
+  function handleButtonClick (button, row, col) {
+    if (selectedQueens.length < 2) {
+      selectedQueens.push([row, col])
+      button.classList.add('queen', 'selected')
+      button.innerHTML = '&#9813;' // Unicode for chess queen
```
Comment (DanCarl857) [inline assets/main.js:37]: Love the use of unicode here

File: assets/main.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      selectedQueens = [[row, col]]
+      button.classList.add('queen', 'selected')
+      button.innerHTML = '&#9813;'
+    }
+  }
+
+  function checkQueens () {
+    if (selectedQueens.length === 2) {
+      const [queen1, queen2] = selectedQueens
+      const result = canQueensAttack(queen1, queen2)
```
Comment (DanCarl857) [inline assets/main.js:54]: Love your use of functions

**Review by DanCarl857 — APPROVED** [review]
Good stuff

File: assets/main.js (hunk 1 above)
Comment (ndikintum) [reply assets/main.js:11]: Sir, how do is make it better

**Pull request merged**